gcloud/connection.py

```diff
@@ -391,7 +391,8 @@ def api_request(self, method, path, query_params=None,
             target_object=_target_object)
 
         if not 200 <= response.status < 300:
-            raise make_exception(response, content)
+            raise make_exception(response, content,
+                                 request_string=method+' '+url)
```

Comment:
I thought PEP8 said no spaces around operators in arguments (or maybe they just meant around the `=` sign...)